src/tictactoe/test/tictactoe_web/game_controller/start_game_test.exs

```diff
@@ -0,0 +1,13 @@
+defmodule TictactoeWeb.GameControllerTest do
+  use TictactoeWeb.ConnCase
+
+  test "POST /api/start_game/", %{conn: conn} do
+    game_id = "id-to-create-game"
+    conn = post(conn, "/api/start_game", %{game_id: game_id})
+
+    actual = json_response(conn, 200)["game_id"]
+
+    assert actual == game_id
```

Comment:
I'd prefer to see 
```
assert %{"game_id" => game_id } == actual
```

It reveals the structure and how to grow it in the future.
But this is more of a nitpick.

Reply:
Or do the JSON file trick you did underneath, but I assume the id is random?

Reply:
The JSON-trick is not a good match here, since the generated value is important.
Agree with more structure-revealing notation.
Why not `actual == %{"game_id" => game_id }` ? Is this order somehow functional tests specific?

Reply:
No, the order doesn't matter. Elixirs exUnit is friendly enough to give you a
```
left: ...
right: ...
```
output so you'll be able to quickly tell which is which 👍